app/templates/association/recognition/index.hbs

```diff
@@ -66,7 +66,7 @@
       />
     </c.header>
     {{#if this.hasErrored}}
-      <TableMessage::Error />
+      {{!-- <TableMessage::Error /> --}}
```

Comment:
@Andresdev02 The `<TableMessage>` and `<TableMessage::Error>` components don't seem to exist in the app. Does this ring a bell? Were they removed at some point?

I think these error states are "broken" at the moment, or does the app simply never get into those states?

Not sure if we should re-add them, or remove the conditional blocks. Any insights?

Embroider is more strict than the legacy build system, so it fails the build if it can't find components that are in use somewhere. (The legacy build would throw an error at runtime).